### PR TITLE
PR, Commenting and Testing Modifications to CONTRIBUTING

### PR DESCRIPTION
**Pull request by parfei**
# Description
This PR fixes https://github.com/Mafia-Hands/Mafia/issues/105

Added guidelines in `CONTRIBUTING` about:
*  PRs require 2+ reviewers
*  PR length
*  Link code conventions
* Commenting in the codebase
* Testing in the codebase

# Demo
image

# Type of change:
Documentation

# Checklist:
* Have you merged main into your branch?
* Have you tested your changes to ensure it works as expected and does not break existing functionality?
* If applicable, please ensure sufficient tests are added that is related to the changes.
* Please ensure this PR has a `label`, is linked to an `issue` and is related to a `project`
* If new documentation is required for this change, have you created a new documentation issue that describes the documentation needed?

### CODE REVIEW

**Review by UltimaKuma — APPROVED** [review]
LGTM

**Review by SreenikethRaghavan — APPROVED** [review]
Great documentation changes @parfei 👍

**Review by jteo97 — APPROVED** [review]
Good additions Jennifer

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,20 +60,40 @@ Refactoring:
...
+
 For a new feature request, the issue should describe the new feature and why it is needed.  
 For bug reports, provide as much information as you can so that the person who fixes the bug will be able to reproduce it.  
+
 For all new issues, please make sure you check the open issues first to avoid creating duplicate issues.  
 If more than one person works on an issue, ensure this is documented in the issue and pull request comments.
+
+## Tests
+All  additions or modifications to code should include associated tests (an automated test suite should be created).
+Please see [Puppeteer](https://github.com/Mafia-Hands/Mafia/wiki/Frontend-Tech) and [backend testing](https://github.com/Mafia-Hands/Mafia/wiki/Backend-Testing-Infrastructure) for more details on how to create tests within our codebase.
```
Comment (UltimaKuma) [inline CONTRIBUTING.md:89]: The puppeteer link is not functioning. Maybe just link to the Frontend page or the puppeteer markdown file?

**Review by jteo97 — APPROVED** [review]

**Review by UltimaKuma — CHANGES_REQUESTED** [review]
LGTM, just one small change to make regarding a broken link.

File: CONTRIBUTING.md (hunk 1 above)
Comment (parfei) [reply CONTRIBUTING.md:89]: Fixed, thanks :)

**Review by UltimaKuma — APPROVED** [review]
Nice!

**Pull request merged**